presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/ScalarImplementation.java

```diff
@@ -217,6 +220,11 @@ public boolean isNullable()
         return nullFlags;
     }
 
+    public Map<Integer, Class> getLambdaInterface()
```

Comment:
Make this take an `int parameterIndex`. Add a `TODO` to `nullFlags` and `nullableArguments`